src/main/java/com/minecolonies/blockout/views/DropDownList.java

```diff
@@ -0,0 +1,308 @@
+package com.minecolonies.blockout.views;
+
+import com.minecolonies.blockout.controls.Button;
+import com.minecolonies.blockout.controls.ButtonVanilla;
+import com.minecolonies.blockout.controls.Label;
+import com.minecolonies.blockout.views.ScrollingList;
+import com.minecolonies.blockout.OverlayView;
+import com.minecolonies.blockout.Pane;
+import com.minecolonies.blockout.PaneParams;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * A DropDownList is a Button which when click display a ScrollingList below it.
+ */
+//TODO do not extend ButtonVanilla but extend view
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)